SofaScoreAcademy/SofaScoreAcademy/ViewController.swift

```diff
@@ -9,98 +9,78 @@ import UIKit
+            Event(teamOneName: "Manchester United", teamTwoName: "Barcelona", startTime: startTime4, matchStatus: .notStarted)
         ]
         
     }
-    
+
     private func setupView() {
         
         view.backgroundColor = .lightGray
-        view.addSubview(stackView)
+        view.addSubview(tableView)
+
+        tableView.delegate = self
+        tableView.dataSource = self
+        tableView.register(MatchViewCell.self, forCellReuseIdentifier: "MatchViewCell")
         
     }
 
-    func addViews() {
+    func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
         
-        for event in exampleEvents {
-            
-            let matchView = MatchView()
-            matchView.configure(with: event)
-            stackView.addArrangedSubview(matchView)
-            
-        }
+        return exampleEvents.count
         
-        let leagueView = LeagueView()
-        let exampleLeague = League(country: "Spain", leagueName: "LaLiga")
-        leagueView.configure(with: exampleLeague)
-        view.addSubview(leagueView)
     }
-    
-    func styleViews() {
+
+    func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
         
-        stackView.axis = .vertical
-        stackView.distribution = .fillEqually
-        stackView.alignment = .fill
-        stackView.spacing = 10
+        guard let cell = tableView.dequeueReusableCell(withIdentifier: "MatchViewCell", for: indexPath) as? MatchViewCell else {
+            return UITableViewCell()
+        }
+        let event = exampleEvents[indexPath.row]
+        cell.configure(with: event)
+        return cell
         
     }
-    
-    func setupConstraints() {
-        
-        stackView.snp.makeConstraints {
-            $0.centerX.centerY.equalToSuperview()
-            $0.width.equalTo(360)
-            
-        }
+
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
         
+        return 56
         
-        if let leagueView = view.subviews.first(where: { $0 is LeagueView }) {
-            
-            leagueView.snp.makeConstraints {
-                
-                $0.bottom.equalTo(stackView.snp.top).offset(-56)
-                $0.centerX.equalToSuperview()
-                $0.width.equalTo(stackView.snp.width)
-                
-            }
-        }
+    }
+
+    private func setupConstraints() {
         
-        stackView.arrangedSubviews.forEach {
-            
-            if let matchView = $0 as? MatchView {
-                
-                matchView.snp.makeConstraints {
-                    $0.height.equalTo(46)
-                    
-                }
-            }
+        tableView.snp.makeConstraints { make in
+            make.center.equalToSuperview()
+            make.height.equalTo(224).priority(.low)
+            make.width.equalToSuperview()
+
         }
```

Comment:
ja bi radije da ti das tableView da bude preko cijelog ekrana to mi nekak ima vise smisla ili da bude ono neki padding od rubova bit ce ljepse